pylint/config/environement_variable.py

```diff
@@ -0,0 +1,7 @@
+# Licensed under the GPL: https://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+# For details: https://github.com/PyCQA/pylint/blob/main/LICENSE
+# Copyright (c) https://github.com/PyCQA/pylint/blob/main/CONTRIBUTORS.txt
+
+from pylint.config.find_default_config_files import find_pylintrc
```

Comment:
The name of this file is misspelled 😄

Reply:
Do you mean there should not be an "s" ?

Reply:
Isn't it `environment`?

Reply:
Ha right, I thought it was about ``find_default_config_files `` !